frontend/src/metabase/query_builder/components/QueryDownloadPopover/QueryDownloadPopover.tsx

```diff
@@ -1,150 +1,126 @@
+  useKeyPressEvent(
+    e => e.key === "Alt" || e.key === "Option",
+    () => {
+      toggleAltPressed(true);
+    },
+    () => {
+      toggleAltPressed(false);
+    },
+  );
 
   return (
-    <DownloadPopoverRoot isExpanded={hasTruncatedResults}>
-      <DownloadPopoverHeader>
-        <h4>{t`Download full results`}</h4>
-        <Tooltip label={formattingInfoTooltipLabel}>
+    <Stack w={hasTruncatedResults ? "18.75rem" : "16.25rem"} py="sm">
+      <Group align="center" position="apart" px="sm">
+        <Title order={4}>{t`Download full results`}</Title>
+        <Tooltip label={getFormattingInfoTooltipLabel()}>
           <Icon name="info_filled" />
         </Tooltip>
-      </DownloadPopoverHeader>
+      </Group>
+
       {hasTruncatedResults && (
-        <DownloadPopoverMessage>
+        <Text px="sm">
           <div>{t`Your answer has a large number of rows so it could take a while to download.`}</div>
           <div>{limitedDownloadSizeText}</div>
-        </DownloadPopoverMessage>
+        </Text>
       )}
-      {formats.map(format => (
-        <DownloadButton
-          key={format}
-          format={format}
-          hasUnformattedOption={checkCanManageFormatting(format)}
-          isHoldingAltKey={isHoldingAltKey}
-          onDownload={handleDownload}
-        />
-      ))}
-    </DownloadPopoverRoot>
+
+      <Stack spacing="sm">
+        {formats.map(format => {
+          return (
+            <DownloadButton
+              key={format}
+              format={format}
+              onClick={() => {
+                onDownload({
+                  type: format,
+                  enableFormatting: !(
+                    checkCanManageFormatting(format) && isAltPressed
+                  ),
+                });
+              }}
+              isAltPressed={isAltPressed}
+            />
+          );
+        })}
+      </Stack>
+    </Stack>
   );
 };
 
-interface DownloadButtonProps {
-  format: string;
-  hasUnformattedOption: boolean;
-  isHoldingAltKey: boolean;
-  onDownload: (format: string, enableFormatting: boolean) => void;
-}
-
 const DownloadButton = ({
   format,
-  hasUnformattedOption,
-  isHoldingAltKey,
-  onDownload,
-}: DownloadButtonProps) => {
-  const { hovered, ref } = useHover<HTMLButtonElement>();
+  onClick,
+  isAltPressed,
+}: {
+  format: typeof exportFormats[number] | typeof exportFormatPng;
+  onClick: () => void;
+  isAltPressed: boolean;
+}) => {
+  const { hovered, ref } = useHover();
+
+  const showUnformattedInfo =
+    hovered && checkCanManageFormatting(format) && isAltPressed;
 
   return (
-    <DownloadButtonRoot
-      onClick={() => onDownload(format, !isHoldingAltKey)}
+    <Group
+      py="xs"
+      px="sm"
+      bg={hovered ? "brand" : "bg-white"}
+      c={hovered ? "text-white" : "text-dark"}
+      fw="bold"
+      position="apart"
+      onClick={onClick}
       ref={ref}
+      style={{ cursor: "pointer", borderRadius: "0.5rem" }}
```

Comment:
I'm still thinking about how to write this code more elegantly. I hate the fact that there's a `hover` state that's not CSS, so I'm thinking of having a CSS module with these styles (instead of a bunch of ternaries and this `style` tag) and putting a `&:hover` with a display property for `(Unformatted)`